js/widgets/rhythmruler.js

```diff
@@ -244,6 +245,16 @@ class RhythmRuler {
                     this._dissectNumber.value.length - 1
                 );
             }
+            if (event.keyCode === RhythmRuler.BACK) {
+                if (this._dissectNumber.value.length == 1) {
+                    this._dissectNumber.value = "";
+                } else {
+                    this._dissectNumber.value = this._dissectNumber.value.substring(
```

Comment:
Isn't this dependent upon the cursor position in the string?

Reply:
@walterbender, I have made the necessary changes to incorporate your suggestions.